doc/api/util.md

```diff
@@ -10,6 +10,45 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+Takes an `async` function (or a function that returns a Promise) and returns a
```

Comment:
I would probably start this with "This method converts promise returning APIs to callback taking ones" or something super explicit at the start.